app/static/js/app.js

```diff
@@ -273,11 +259,11 @@ $( document ).ready(function() {
 });
 
 function hoverInLogo(hoveredOver){
-	document.getElementById("active").id="oldActive";
+	//document.getElementById("active").id="oldActive";
```

Comment:
Yeah you can merge it darrel. The line I commented out does not cause the system to crash or anything, it just throws an annoying message in the console